src/gui/sdlmain.cpp

```diff
@@ -3890,7 +3740,7 @@ bool GFX_Events()
 				ApplyInactiveSettings();
 				GFX_LosingFocus();
 				CPU_Enable_SkipAutoAdjust();
-				sdl.mouse.has_focus = false;
+				MOUSE_NotifyFocus(false);
```

Comment:
I'm struggling here with the wording.

`has_focus` is very self-documenting: the mouse either has or doesn't have focus.

But 'NotifyFocus(bool)` doesn't tell what's going on anymore.

Are we setting focus on/off?
Are we setting notifications on/off?
Are we focusing notifications?

Reply:
Changed to `MOUSE_NotifyHasFocus` - I hope the purpose is more clear now.

Reply:
`MOUSE_NotifyHasFocus` -> I think the rewording helps a lot - thanks! My understanding is that, if we generate a notification message, for example, "You can uncapture the mouse with middle-click", then we will also grab/steal mouse-focus when that notigfication happens, so the mouse is active within the DOSBox Staging window?

(Suggest adding an expanded comment like this, because it might not be obvious for newbies in this area like me .. notifications and focus are bit ambiguous how they interplay with each other - even if the function name is very readable).